File: 250905/약수가 세 개인 수/numbers-with-three-divisors.py
start, end = map(int, input().split())

# Please write your code here.
count = 0
for i in range(start, end + 1):
    num = 0
    for j in range(1, start + 1):
        if i % j == 0:
            num += 1
    
    if num == 3:
        count += 1
print(count)

File: 250905/약수가 세 개인 수/numbers-with-three-divisors.js
const fs = require("fs");
let [start, end] = fs.readFileSync(0).toString().trim().split(" ").map(Number);

let count = 0
for (let i = start; i <= end; i ++) {
    let num = 0
    for (let j = 1; j <= i; j++) {
        if (i % j == 0) {
            num += 1;
        }
    }

    if (num == 3) {
        count += 1;
    }
}
console.log(count);